platform/viewer/src/lib/DICOMSR/retrieveDataFromMeasurements.js

```diff
@@ -0,0 +1,52 @@
+import * as dcmjs from 'dcmjs';
+import OHIF from '@ohif/core';
+import cornerstone from 'cornerstone-core';
+
+const retrieveDataFromMeasurements = measurements => {
+  const cornerstoneAdpater = dcmjs.adapters.Cornerstone;
+  const { MeasurementReport } = cornerstoneAdpater;
+  const { getImageIdForImagePath } = OHIF.measurements;
+
+  const toolState = {};
+  const unsupportedTools = [];
+
+  Object.keys(measurements).forEach(measurementType => {
+    const annotations = measurements[measurementType];
+
+    annotations.forEach(annotation => {
+      const { toolType, imagePath } = annotation;
+      if (cornerstoneAdpater[toolType]) {
```

Comment:
It might be dangerous to assume a string name guarantees a match.

Reply:
We can think on something stronger, but for now, on dcmjs its the only way I thought. 

@JamesAPetts do you have any idea if we can check this in a better way? If a tool is supported by dcmjs?